site/js/content/index.js

```diff
@@ -69,12 +69,12 @@ content.use((state, emitter) => {
       state.ws.removeListener('edited chat message', handleEditMessage)
     }
 
-    // establish/use websocket
-    state.ws = new Ws(host)
-
-    // listen for events
-    state.ws.on('received chat message', handleNewMessage)
-    state.ws.on('edited chat message', handleEditMessage)
+    if (host) {
+      // establish/use websocket
+      state.ws = new Ws(host)
+      state.ws.on('received chat message', handleNewMessage)
+      state.ws.on('edited chat message', handleEditMessage)
+    }
```

Comment:
This was causing an error when you navigated to the page without initially giving a server path (i.e. `localhost:3000`, not `localhost:3000/my.cool.server:12345`) so I fixed it.